third_party/silabs/efr32_sdk.gni

```diff
@@ -191,6 +204,16 @@ template("efr32_sdk") {
       #"__STACK_SIZE=0",
     ]
 
+    if (chip_enable_ble_rs911x) {
```

Comment:
Those two define have the same if condition. Don't split it in two if. regroup them.

Also why do you need `RSI_BLE_ENABLE`=1 and `RS91X_BLE_ENABLE`=1 why do you need two defines that are effectively the same

Reply:
we will combine both "if" conditions.
RSI_BLE_ENABLE is used in the SAPI code of wiseconnect to enable the BLE on RS9116. The macro RS91X_BLE_ENABLE is used on the matter side examples/platform code separately because if someone changed the macro on SAPI we should not need to do matter side code changes only need to change the macro definition.

Reply:
I would advocate to actually try to reuse the SAPI defines. Even if they change in SAPI, you aren't updating the sdk(wiseconnect) that often. If you do update it, you would go over test and validation I assume and can catch the change and address it in your update sdk pr. 


I am saying this because we are starting to have so many defines and args that it is hard to follow. Same thing for our thread platform by the way. If we can share defines from gsdk to our matter platform code we should